integrations/pinecone/tests/test_document_store.py

```diff
@@ -257,3 +260,59 @@ def test_embedding_retrieval(self, document_store: PineconeDocumentStore):
         assert len(results) == 2
         assert results[0].content == "Most similar document"
         assert results[1].content == "2nd best document"
+
+    def test_sentence_window_retriever(self, sleep_time, document_store: PineconeDocumentStore):
```

Comment:
You don't need `sleep_time`. It should be automatically applied when using the `PineconeDocumentStore` fixture.
https://github.com/deepset-ai/haystack-core-integrations/pull/1034#discussion_r1735809950